frontend/tests/unit/tests/services/query-service-test.js

```diff
@@ -46,8 +47,11 @@ describe('QueryService', function() {
     $provide.constant('ConfigurationService', configurationService);
   }));
 
-  beforeEach(inject(function(_QueryService_){
+  beforeEach(inject(function(_QueryService_, _$rootScope_, _$httpBackend_, _PathHelper_) {
```

Comment:
Identifier '_QueryService_' is not in camel case.<br>Identifier '_$rootScope_' is not in camel case.<br>Identifier '_$httpBackend_' is not in camel case.<br>Identifier '_PathHelper_' is not in camel case.